src/noisepy/seis/pnwstore.py

```diff
@@ -101,12 +101,16 @@ def read_data(self, timespan: DateTimeRange, chan: Channel) -> ChannelData:
             f"FROM tsindex WHERE network='{chan.station.network}' AND station='{chan.station.name}' "
             f"AND channel='{chan.type.name}' and location='{chan.station.location}' "
             f"AND filename LIKE '%%/{chan.station.network}/{year}/{doy}/%%'"
+            "ORDER BY byteoffset ASC"
```

Comment:
@niyiyu does this need a space at the end of the previous line? `.../%%' "`

Reply:
Oh seems I missed the space here. However, they seem giving the same results : `.../%%'ORDER BY`..😂
Do you think it worth another PR to add this space?